src/components/SignupsSplit.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTreatments, SplitContext } from '@splitsoftware/splitio-react';
+
+const SignupsSplit = ({
+  testChildren,
+  defaultChildren,
+  treatmentWithConfig,
+}) => {
+  const treatments = useTreatments(['signups']);
+  const { treatment } = treatmentWithConfig;
```

Comment:
```suggestion
  const { treatment, config } = treatments[testName];
```

Reply:
This is similar to what's going on here https://github.com/newrelic/gatsby-theme-newrelic/pull/503/files#diff-c3e9cf10d34504caf408d39e6042ae504aaa642bba82ab1778b62109e771ba8eL38-L44